ethers-core/src/types/chain.rs

```diff
@@ -47,6 +47,7 @@ pub type ParseChainError = TryFromPrimitiveError<Chain>;
 #[strum(serialize_all = "kebab-case")]
 #[repr(u64)]
 pub enum Chain {
+    #[strum(serialize = "ethlive")]
```

Comment:
strum is weird and you have to this

```suggestion
    #[strum(to_string = "mainnet", serialize = "ethlive")]
```

Reply:
That's right, the proposed PR breaks the string representation. I've fixed it by adding another `serialize` parameter, it matches what other enum variants do. I've added a test, checking that, but I see that in #2270 you've solved the tests coverage once and for all, so I'll remove them.

Reply:
Maybe it makes sense to switch multiple `serialize` parameters with implicitly the last one being the `to_string` representation with the explicit `to_string` parameters?